examples/volumes/azure_disk/claim/README.md

```diff
@@ -0,0 +1,94 @@
+# Kubernetes Persistent Volume Plugin for Blob and Managed Disks Samples
+
+This repo contains samples that works with the new Azure persistent volume plugin for Kubernetes. The plugin is expected to be in v1.8 release then will become part of Azure ACS 
```

Comment:
1.8?
I thought it already landed in 1.7.2.
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md#changelog-since-v171
See entry 3 `Azure PD (Managed/Blob) (#46360, @khenidak)`